doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,148 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
```

Comment:
This sentence is confusing (we just read about "the solution of the biharmonic equation" and now we have "the *basic* solution of the biharmonic *function*" --- should be "equation" again) and it's not clear what it's adding to the message we're trying to convey... Maybe "in [3], the solution is generalized(?)" and we're basing our implementation (or this example?) on this reference??